config/common.php

```diff
@@ -29,6 +25,9 @@
 }
 
 return [
+    'params' => static function () use ($params) {
```

Comment:
@samdark we need to make some hack (an interface or other) to pass the parameters to the application.
AFAIK it's at least 5 attempt that could be reused.